.gitlab-ci.yml

```diff
@@ -65,6 +65,7 @@ variables:
   BUILD_RASPBERRYPI3: "false"
   BUILD_RASPBERRYPI4: "false"
   CLIENT_ACCEPTANCE_TEST_SPECIFIC: ""
+  CLIENT_ACCEPTANCE_TEST_SUITES: "--platform-tests --software-tests"
```

Comment:
Hmm, initially I was expecting a variable for each. But I guess this is fine.

A bit harder to use only.